shared/src/utils/validators/is-patients-per-uuid.ts

```diff
@@ -0,0 +1,30 @@
+import type { ValidationOptions, ValidationArguments } from 'class-validator';
+import { isUUID } from 'class-validator';
+import type { UUID } from '../uuid';
+import type { UUIDSet } from '../uuid-set';
+import type { PatientCount } from '../../models/radiogram';
+import { createMapValidator } from './create-map-validator';
+import type { GenericPropertyDecorator } from './generic-property-decorator';
+import { makeValidator } from './make-validator';
+import { isPatientCount } from './is-patient-count';
+
+export const isPatientsPerUUID = createMapValidator<UUID, PatientCount>({
```

Comment:
Please use `ResourceDescription<PatientStatus>` instead of `PatientCount` here, too.